_test/test_mpi/test_iMPI_methods.m

```diff
@@ -21,9 +25,21 @@
             obj.current_config_folder = cs.config_folder;
             
             pc = parallel_config;
+            if strcmpi(pc.parallel_framework,'n/a')
```

Comment:
is `n/a` a good constant with a 'special' character in? would `none` be simpler? It's used as a hard-coded string in a *lot* of places